app/views/shared/_fields_for.html.slim

```diff
@@ -0,0 +1,26 @@
+.multiple
+  div id=question.name
+  div id="error_#{f.object.name}_#{question.name}"
+
+  = f.fields_for question.name do |mf|
+    .multiple-wrapper class=(mf.index == 0 ? 'multiple-first' : 'multiple-not-first')
+      = mf.hidden_field :id
+      - question.subquestions.each do |subquestion|
```

Comment:
📓 It's worth pointing out that this block is not using the recursive approach of rendering the questions by using it's mapped display_type. The main reason for not doing that right now, is that, the content for this section follows a very specific pattern that follows out of the one defined in the schema.

Might be something we need to re-visit in the future, for the time being it complies with the existent behaviour, which is what we intended to keep.